src/components/RoomHeaderAvatars.js

```diff
@@ -89,7 +87,6 @@ function RoomHeaderAvatars(props) {
                                 >
                                     <Avatar
                                         source={icon.source}
-                                        fill={themeColors.iconSuccessFill}
```

Comment:
It looks like the intent is to no longer use a `fill` color for avatars. I already asked above, but it would be helpful to understand why we're doing this.